cypress/component/DataAccessRequest/selectable_datasets.spec.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable no-undef */
+import {React} from 'react';
+import {mount} from 'cypress/react';
+import DataAccessRequestApplication from '../../../src/pages/dar_application/DataAccessRequestApplication';
+import SelectableDatasets from '../../../src/pages/dar_application/SelectableDatasets.jsx';
+
+const props = {
+  datasets: [
+    {
+      dataSetId: 123456,
+      datasetIdentifier: `DUOS-123456`,
+      datasetName: 'Some Dataset 1'
+    },
+    {
+      dataSetId: 234567,
+      datasetIdentifier: `DUOS-234567`,
+      datasetName: 'Some Dataset 2'
+    },
+    {
+      dataSetId: 345678,
+      datasetIdentifier: `DUOS-345678`,
+      datasetName: 'Some Dataset 3'
+    },
+    {
+      dataSetId: 456789,
+      datasetIdentifier: `DUOS-456789`,
+      datasetName: 'Some Dataset 4'
+    },
+  ], 
+  setSelectedDatasets: () => {},
+  disabled: false
+};
+
+const propsDisabled = {
+  datasets: [
+    {
+      dataSetId: 123456,
+      datasetIdentifier: `DUOS-123456`,
+      datasetName: 'Some Dataset 1'
+    },
+    {
+      dataSetId: 234567,
+      datasetIdentifier: `DUOS-234567`,
+      datasetName: 'Some Dataset 2'
+    },
+    {
+      dataSetId: 345678,
+      datasetIdentifier: `DUOS-345678`,
+      datasetName: 'Some Dataset 3'
+    },
+    {
+      dataSetId: 456789,
+      datasetIdentifier: `DUOS-456789`,
+      datasetName: 'Some Dataset 4'
+    },
+  ], 
+  setSelectedDatasets: () => {},
+  disabled: true
```

Comment:
To avoid duplication, you could construct a `const datasets` object, and then create a `props` and `propsDisabled` object from that dataset object.